src/content/app/species-selector/state/speciesSelectorEpics.ts

```diff
@@ -64,10 +64,16 @@ export const ensureCommittedSpeciesEpic: Epic<Action, Action, RootState> = (
       const { genomeInfo } = action.payload;
```

Comment:
If I understand correctly, at the moment, in this branch, the payload from `fetchGenomeInfo` will not contain the fields expected below. It is still using the old endpoint from the genome search api, whose payload can be seen e.g. [here](https://beta.ensembl.org/api/genomesearch/genome/info?genome_id=a7335667-93e7-11ec-a39d-005056b38ce3).

The endpoint is being replaced in https://github.com/Ensembl/ensembl-client/pull/1047, after which we should have access to these fields.